apps/sasquatch/build.gradle

```diff
@@ -75,21 +75,17 @@ dependencies {
     //noinspection GradleDependency
     jcenterDependencyImplementation "com.microsoft.appcenter:appcenter-push:${appCenterSdkVersion}"
 
-    firebaseImplementation "com.google.firebase:firebase-core:${rootProject.ext.firebaseLibVersion}"
-    firebaseImplementation "com.google.firebase:firebase-messaging:${rootProject.ext.firebaseLibVersion}"
+    firebaseImplementation "com.google.firebase:firebase-core:16.0.1"
```

Comment:
We can TODO remove this after SDK available in jCenter. Can also add jCenter to the directive for now.

Reply:
I tried it before but in fact `gradle` cannot handle it for some reason :confused:
`> Could not find method jcenterDependencyFirebaseImplementation()`

Reply:
Let's just add a TODO then.